src/shared/cryptsetup-util.h

```diff
@@ -7,6 +7,15 @@
 #if HAVE_LIBCRYPTSETUP
 #include <libcryptsetup.h>
 
+/* These next two are defined in libcryptsetup.h from cryptsetup version 2.3.4 forwards. */
+#ifndef CRYPT_ACTIVATE_NO_READ_WORKQUEUE
+#define CRYPT_ACTIVATE_NO_READ_WORKQUEUE (1 << 24)
+#endif
+#ifndef CRYPT_ACTIVATE_NO_WRITE_WORKQUEUE
+#define CRYPT_ACTIVATE_NO_WRITE_WORKQUEUE (1 << 25)
+#endif
+/* End of backwards compatibility defines. */
```

Comment:
We don't usually add this kind of comments after backward compat defines

Reply:
Want me to remove it?

Reply:
If you could, that would be grand. Thanks!

Reply:
OK, removed.